ui/components/app/set-approval-for-all-warning/set-approval-for-all-warning.js

```diff
@@ -65,12 +64,13 @@ const SetApproveForAllWarning = ({
           name={IconName.Danger}
           className="set-approval-for-all-warning__content__header__warning-icon"
         />
-        <Typography
-          variant={TypographyVariant.H4}
-          fontWeight={FONT_WEIGHT.BOLD}
+        <Text
+          variant={TextVariant.headingSm}
+          as="h4"
+          fontWeight={FontWeight.Bold}
```

Comment:
Since `headingSm` has tag h4 and font-weight already added to it, you can remove it
```suggestion
        <Text
          variant={TextVariant.headingSm}
```